synapse/handlers/room.py

```diff
@@ -425,17 +425,19 @@ async def clone_existing_room(
 
         # Copy over user power levels now as this will not be possible with >100PL users once
         # the room has been created
-
         # Calculate the minimum power level needed to clone the room
         event_power_levels = power_levels.get("events", {})
-        state_default = power_levels.get("state_default", 0)
-        ban = power_levels.get("ban")
+        state_default = power_levels.get("state_default", 50)
+        ban = power_levels.get("ban", 50)
         needed_power_level = max(state_default, ban, max(event_power_levels.values()))
 
+        # Get the user's current power level, this matches the logic in get_user_power_level,
+        # but without the entire state map.
+        current_power_levels = power_levels.setdefault("users", {})
+        current_power_level = current_power_levels.get(user_id, 0)
```

Comment:
Oh hmmm this should default to `users_default` probably?

Reply:
Oh, yes indeed. `get_user_power_level` only defaults to `0` if `users_default` is not set.

Reply:
I made this change in 305762d, hopefully it sitll looks good! 👍

Reply:
FTR I think all this would do is cause some people's power levels to get increased in unnecessary situations. But since they get reset anyway it isn't a huge deal.